kolii_task1.py

```diff
@@ -0,0 +1,8 @@
+line = 'Vladyslav Ovsianikov, ' * 45
+result = ((15 + 11.1/3) / (33 + 12.3)) - 123.1
+result = round(result, 3)
+print('Multi-paradigm languages: Task 1')
+print('Andrii Kolii FI-9119')
+print(line[:-2])
+print(result)
```

Comment:
please add the blank lines in your code.
you can use them to divide the code for logic blocks.
For instance, you can divide this code for block of preparations and block of prints.